src/main/java/data/Camera.java

```diff
@@ -55,7 +55,18 @@ public Camera(String name, String description, CameraType cameraType) {
         log.debug("Created new Camera(name={}, description={}, cameraType={}",
                 name, description, cameraType);
     }
-
+    
+    /**
+     * Clone constructor.
+     * @param camera camera to clone.
+     */
+    public Camera(Camera camera) {
```

Comment:
Take a look at the [Cloneable](https://docs.oracle.com/javase/7/docs/api/java/lang/Cloneable.html) interface